src/components/navigation/navigation.scss

```diff
@@ -96,12 +96,12 @@ header {
 .powered-by {
     position: absolute;
     bottom: 0;
+    left: 0;
     font-size: pxToRem(12);
     padding: pxToRem(8) pxToRem(12);
     background-color: rgba(var(--kompendium-contrast-100), 0.4);
     backdrop-filter: blur(pxToRem(4));
     width: 100%;
-    margin-left: pxToRem(-4);
```

Comment:
<details><summary>Nice mode</summary><br>
Nice! Good work! 👍
</details>

<details><summary>Nitpick mode</summary><br>
This is clearly an improvement, but the background still overlaps the navbar's gutters in a way I don't like:<br>
<br>
<img width="341" alt="image" src="https://user-images.githubusercontent.com/154725/212702224-7e185383-b0df-4317-a991-4ecbaf0fb07c.png"><br>
<img width="341" alt="image" src="https://user-images.githubusercontent.com/154725/212702180-9fc36f32-a0ca-4ca5-8bbd-cdcc6cb9b347.png">

Reply:
Thanks Adrian. But that is intentional actually 😸 
Can you please elaborate one "a way I don't like" 😅

Reply:
I think it's because it's such a subtle difference in colour that it just looks wrong… 🤷‍♂️

Since it's on purpose, I'll merge this 😊